specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2022-10-01-preview/machineLearningServices.json

```diff
@@ -5935,6 +5928,63 @@
       "additionalProperties": false,
       "x-ms-discriminator-value": "UsernamePassword"
     },
+    "WorkspaceConnectionAccessKey": {
+      "type": "object",
+      "properties": {
+        "accessKeyId": {
+          "type": "string"
+        },
+        "secretAccessKey": {
```

Comment:
>secretAccessKey [](http://example.com/codeflow?start=9&length=15)

If this is a secret, it needs to be tagged with the x-ms-secret extension. Also, secrets are not allowed in GET payloads. Does your service null/blank out this property on GET? Ref: [https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-secret](https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-secret). Ref: [https://armwiki.azurewebsites.net/rp_onboarding/process/api_review_best_practices.html?q=secret%20POST#common-issues-found-in-review](https://armwiki.azurewebsites.net/rp_onboarding/process/api_review_best_practices.html?q=secret%20POST#common-issues-found-in-review).